src/cmds/docs/cleanup.ts

```diff
@@ -0,0 +1,102 @@
+import type { CommandOptions } from '../../lib/baseCommand';
+
+import chalk from 'chalk';
+import config from 'config';
+
+import Command, { CommandCategories } from '../../lib/baseCommand';
+import createGHA from '../../lib/createGHA';
+import deleteDoc from '../../lib/deleteDoc';
+import getDocs from '../../lib/getDocs';
+import promptTerminal from '../../lib/promptWrapper';
+import readdirRecursive from '../../lib/readdirRecursive';
+import readDoc from '../../lib/readDoc';
+import { getProjectVersion } from '../../lib/versionSelect';
+
+export type Options = {
+  dryRun?: boolean;
+  folder?: string;
+};
+
+function getSlug(filename: string): string {
+  const { slug } = readDoc(filename);
+  return slug;
+}
+
+export default class DocsCleanupCommand extends Command {
+  constructor() {
+    super();
+
+    this.command = 'docs:cleanup';
```

Comment:
Thoughts on renaming this to `docs:prune`?

Reply:
Love it!

Reply:
Done: https://github.com/readmeio/rdme/pull/644/commits/863d06cdbeedf06a3cc2c6473f42035a04b8609f